using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -50,7 +121,11 @@ On iOS, UUID is persisted in `NSUserDefaults`.
                        callback: (nullable void (^) (NSDictionary* __nullable visitorProfile)) callback;
 ```
 
-#### **Examples**
+o  `data` is the traits data for the current visitor
```

Comment:
- Add a period at the end.
- Remove "o" and replace with an asterisk for the bullet point, so that it reads like this:
"* `data` is the traits data for the current visitor."